Index: app/Http/Controllers/Admin/LessonController.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/Http/Controllers/Admin/LessonController.php	(revision 2db98f4d62697ffe774b1ee0b84d62f17883492c)
+++ app/Http/Controllers/Admin/LessonController.php	(date 1515928148833)
@@ -35,9 +35,21 @@
      * @param  \Illuminate\Http\Request $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(Request $request, Lesson $lesson)
     {
-        //
+        $lesson['title'] = $request['title'];
+        $lesson['introduce'] = $request['introduce'];
+        $lesson['preview'] = $request['preview'];
+        $lesson['iscommend'] = $request['iscommend'];
+        $lesson['ishot'] = $request['ishot'];
+        $lesson['click'] = $request['click'];
+        $lesson->save();
+
+        $videos = json_decode($request['videos'], true);
+        foreach ($videos as $video) {
+            $lesson->videos()->create($video);
+        }
+        return redirect('/admin/lesson');
     }
 
     /**
Index: app/Model/Lesson.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/Model/Lesson.php	(revision 2db98f4d62697ffe774b1ee0b84d62f17883492c)
+++ app/Model/Lesson.php	(date 1515927477938)
@@ -8,4 +8,13 @@
 {
     //不允许添加的字段
     protected $guarded = [];
+
+    /**
+     * 与视频模型关联
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function videos()
+    {
+        return $this->hasMany(Video::class);
+    }
 }
